src/site/stages/build/process-cms-exports/schemas/raw/node-health_care_region_detail_page.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase */
+
+module.exports = {
+  type: 'object',
+  properties: {
+    title: { $ref: 'GenericNestedString' },
+    changed: { $ref: 'GenericNestedString' },
+    path: { $ref: 'RawPath' },
+    field_alert: { $ref: 'GenericNestedString' },
+    field_content_block: { $ref: 'EntityReferenceArray' },
+    field_featured_content: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
+    field_intro_text: { $ref: 'GenericNestedString' },
+    field_office: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
+    field_related_links: {
+      type: 'array',
+      maxItems: 1,
+      items: { $ref: 'EntityReference' },
+    },
+    field_table_of_contents_boolean: { $ref: 'GenericNestedBoolean' },
```

Comment:
Data type in the name. Cool. Any idea what it does, out of curiosity?

Reply:
It looks like it [controls whether or not some DOM nodes get rendered](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/layouts/health_care_region_detail_page.drupal.liquid#L29-L34), and I'm hoping that something else comes along later to fill in the details.